src/ModRamFunctions.f90

```diff
@@ -723,4 +510,50 @@ subroutine test_Gcoul(verbose)
 
   end subroutine test_Gcoul
 
+  subroutine test_NewtFit(verbose)
```

Comment:
I'm wondering if we should just get rid of the NewtFit functions completely and drop this test. With the GSL interpolation functions we don't actually use this anywhere.

Reply:
I'm happy to add paring down the codebase to drop obsolete or redundant code as a task for the next release (or maybe the one after, as I want to get to doing numbered versions more frequently). For now this doesn't really impact anything beyond showing that a function we don't use any more has expected behavior...